### ISSUE TITLE
Card: Badge information ‘New’ as highlighted in red is not conveyed along with the card information, when accessed.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3817]
Description: Badge information ‘New’ as highlighted in red is not conveyed along with the card information, when accessed.

Expected: All the information associated with the UI element should be conveyed by the screen reader.

Screenshot: 
![image](https://user-images.githubusercontent.com/32538291/98773004-02cc0800-240e-11eb-8d06-5c1be49467fe.png)
